zerver/tests/test_outgoing_webhook_interfaces.py

```diff
@@ -48,6 +48,16 @@ def make_response(text: str) -> requests.Response:
             )
         self.assertTrue(m.called)
 
+        response = make_response(text=json.dumps(dict(content="")))
```

Comment:
I might write a brief comment above this, like "# Verify that we don't try to generated a message with empty content"

Reply:
(If it were its own test function, we'd probably have the test function name basically explain that goal, but as a piece within a larger test, it's worth documenting what each chunk is doing).  Feel free to add additional comments to existing test that you had to spend time figuring out when reading it to extend the test, as well -- our goal is for tests to be quickly readable for folks without a lot of context!